### ISSUE TITLE
Error: add_() takes 1 positional argument but 2 were given

### ISSUE THREAD

**Issue by Madhura12gj** [OpenMined/PySyft#3562]
Hi there, I'm trying out FL on the wisonsin breast cancer dataset, and I get the following error while I try to train.
 `Error : add_() takes 1 positional argument but 2 were given`
Here's a snippet of the training function, any help on why this occurs would be appreciated. Let me know if anything else is needed.
`def train(args, model, device, train_loader, optimizer, epoch):`
    `model.train()`
    `for batch_idx, (inputs, targets) in enumerate(federated_train_loader):` # <-- now it is a distributed   
       dataset
        `model.send(inputs.location)` # <-- NEW: send the model to the right location
        `inputs, targets = inputs.to(device), targets.to(device)`
        `optimizer.zero_grad()`
        `output = model(inputs.float())`
        `loss = criterion(output, targets.float())`
        `loss.backward()`
        `optimizer.step()`
        `model.get()` # <-- NEW: get the model back
        `if (batch_idx % args.log_interval) == 0:`
          `loss_int = loss.get()`
`device = torch.device("cpu")`
`for epoch in range(1, args.epochs):`
    `train(args, model, device, federated_train_loader, opt, epoch)`

**Comment by tudorcebere** [comment]
Hello! Thank you for reporting this, could you provide:

1. the full script to reproduce the error.
2. the stack trace/logging if you have.
3. syft version and OS.

Thank you!
